pkg/setting/setting_unified_alerting.go

```diff
@@ -388,6 +397,16 @@ func (cfg *Cfg) ReadUnifiedAlertingSettings(iniFile *ini.File) error {
 	}
 	uaCfg.StateHistory = uaCfgStateHistory
 
+	rrWriter := iniFile.Section("unified_alerting.recording_rules.writer")
+	rrWriterProm := iniFile.Section("unified_alerting.recording_rules.writer.prometheus")
+	uaCfgRecordingRules := RecordingRuleSettings{
+		URL:               rrWriter.Key("url").MustString(""),
+		BasicAuthUsername: rrWriter.Key("basic_auth_username").MustString(""),
+		BasicAuthPassword: rrWriter.Key("basic_auth_password").MustString(""),
+		TenantID:          rrWriterProm.Key("tenant_id").MustString(""),
+	}
```

Comment:
should there be some basic validation, e.g. require both user and password, making sure that string is actual URL etc?